api/src/db/models/legacy_mixin/user_mixin.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from sqlalchemy.orm import Mapped, declarative_mixin, mapped_column
+
+
+@declarative_mixin
+class TuserAccountMapperMixin:
+    user_account_id: Mapped[int] = mapped_column(primary_key=True)
+    source_type: Mapped[str]
+    ext_user_id: Mapped[str]
+    ext_issuer: Mapped[str]
+    last_auth_date: Mapped[datetime]
+    created_date: Mapped[datetime | None]
+    creator_id: Mapped[str | None]
+    last_upd_date: Mapped[datetime | None]
+    last_upd_id: Mapped[str | None]
+
+
+@declarative_mixin
+class TuserAccountMixin:
+    user_account_id: Mapped[int] = mapped_column(primary_key=True)
+    user_id: Mapped[str]
+    full_name: Mapped[str | None]
+    email_address: Mapped[str | None]
+    phone_number: Mapped[str | None]
+    first_name: Mapped[str | None]
+    middle_name: Mapped[str | None]
+    last_name: Mapped[str | None]
+    is_deleted_legacy: Mapped[str]
+    is_duplicate: Mapped[str]
+    is_active: Mapped[str]
+    created_date: Mapped[datetime]
```

Comment:
Another mistake of mine - this should be nullable:
```suggestion
    created_date: Mapped[datetime | None]
```